scripts/builtin/glove.dml

```diff
@@ -119,7 +141,7 @@ gloveWithCoocMatrix = function(matrix[double] cooc_matrix, frame[Unknown] cooc_i
     G = cbind(cooc_index[,2], as.frame(G));
 }
 
-glove = function(
+f_glove = function(
```

Comment:
we use the f_ to indicate that this is the public function, therefore we should not modify this line.
the rest of the functions in this file are private, and therefore not available from outside.